summariser/src/main.rs

```diff
@@ -10,6 +10,8 @@ pub(crate) mod filter;
 async fn main() -> anyhow::Result<()> {
     env_logger::init();
 
+    println!("{}", 1e6);
+
```

Comment:
I guess this is leftover from debugging?

```suggestion
```

Reply:
Haha, yes